pkg/reconciler/v1alpha1/taskrun/resources/input_resource_test.go

```diff
@@ -46,20 +47,78 @@ func setUp() {
 		},
 		Spec: v1alpha1.PipelineResourceSpec{
 			Type: "git",
-			Params: []v1alpha1.Param{
-				v1alpha1.Param{
-					Name:  "Url",
-					Value: "https://github.com/grafeas/kritis",
-				},
-			},
+			Params: []v1alpha1.Param{{
+				Name:  "Url",
+				Value: "https://github.com/grafeas/kritis",
+			}},
 		},
 	}
 	pipelineResourceInformer.Informer().GetIndexer().Add(res)
+
+	clusterRes := &v1alpha1.PipelineResource{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "cluster2",
+			Namespace: "marshmallow",
+		},
+		Spec: v1alpha1.PipelineResourceSpec{
+			Type: "cluster",
+			Params: []v1alpha1.Param{{
+				Name:  "Url",
+				Value: "http://10.10.10.10",
+			}},
+			SecretParams: []v1alpha1.SecretParam{{
+				FieldName:  "cadata",
+				SecretKey:  "cadatakey",
+				SecretName: "secret1",
+			}},
+		},
+	}
+	pipelineResourceInformer.Informer().GetIndexer().Add(clusterRes)
+
+	clusterResText := &v1alpha1.PipelineResource{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "cluster3",
+			Namespace: "marshmallow",
+		},
+		Spec: v1alpha1.PipelineResourceSpec{
+			Type: "cluster",
+			Params: []v1alpha1.Param{{
+				Name:  "Url",
+				Value: "http://10.10.10.10",
+			}, {
+				Name: "CAdata",
+				// echo "my-ca-cert" | base64
```

Comment:
might be worth mentioning this in the docs on how to use it

Reply:
it just how you create a secret, which I link to the docs

Reply:
kk